go.mod

```diff
@@ -3,7 +3,7 @@ module terraform-provider-powerstore
 go 1.19
 
 require (
-	github.com/dell/gopowerstore v1.11.1-0.20230320172746-beee61ee87f5
+	github.com/dell/gopowerstore v1.11.1-0.20230331110225-2e4fe0495575
 	github.com/hashicorp/terraform-plugin-docs v0.13.0
 	github.com/hashicorp/terraform-plugin-framework v1.0.1
```

Comment:
Update the plugin framework version as discussed

Reply:
Plugin framework will be taken up as a separate PR